HIP/hip-1081.md

```diff
@@ -0,0 +1,537 @@
+
+## How to Teach This
+
+To effectively educate and inform users about block node operation, comprehensive technical documentation, blogs, and
+webinars will be essential. Technical documentation will provide detailed and in-depth explanations of operation modes,
+usage, and best practices, ensuring that developers and mirror node operators can fully understand and transition to
+block stream consumption from a block node as well as block node hosting.
+
+Blogs will offer more accessible and engaging content, highlighting use cases, real-world applications, and the benefits
+of a block node, catering to a broader audience of Hedera stakeholders. Webinars will serve as interactive platforms
+for live demonstrations, Q&A sessions, and expert insights, enabling participants to gain a deeper understanding through
+direct engagement with subject matter experts.
+
+## Reference Implementation
+
+
+## Rejected Ideas
+
+### BN to CN staking as a requirement for a CN before streaming to a BN.
+It was suggested that to help incentivize long term block nodes that a BN would stake some minimum amount of HBAR to a
+CN. In which upon reward of the full staking period a BN would recuperate staking rewards but also additional reward
+for offering DA
+
+### BN to CN staking as a mechanism to ensure long term DA
+It was suggested that to help incentivize long term block nodes that staked and consumed a stream for at least the
+staking period would receive an additional reward for offering DA
+
+### App to BN allowances to pay for API usage
+It was suggested that one mechanism of paying in advance would be for a client such as a BN or MN or app to first offer
+a BN an allowance of some amount of HBAR. The BN would then periodically debit from the allowance to pay itself after
+responding to a clients calls. The issue was there was a fear that the client to rescind the allowance after time and a
+BN would be 1 or 2 blocks in time off of the accurate allowances which could result in services offered for free if the
+client was being malicious
+
+### BlockItem Service
+Returning a single request BlockItem seemed to not be a valuable service out of the gate. 
+Especially without the ability to cleanly identify an item it doesn’t seem worthwhile for a BN to return a BlockItem vs
+a Block.
+
+### Gossiping Block Node
+<aside>
+🚨 **Open Task:** Summarize initial claims and initial reasons to not support
+</aside>
+
+## Open Issues
+
+- [ ] Q: “As a block node operator I should be able to attach custom plugins to the block node, which can utilize its
+    functionality.” How should this intent be highlighted in specification. Are the detail more implementation and future
+    expansions that should be captured in separate HIPs and product EPICs?
+- [ ] Q: Does the block node require or rely on a transmission of full state from a CN? If so what’s the contract between
+nodes here? Previously the CN output state every 15 mins
```

Comment:
Block Nodes that _manage live state_ or _produce state snapshots_ require a state snapshot to start up if not active since genesis for that network. That snapshot should, however, be transferred manually (either from cloud storage or another block node).

The consensus node, in a block stream mode, should _not ever_ transmit a state snapshot to another system.  State snapshots in this realm are _local backup_ only and should be short-lived ephemeral data (if stored at all).
One of the major benefits of block nodes is that we remove the bandwidth, processing, and complexity of managing state snapshot transfers from consensus nodes entirely.

If a running block node ever needs a state snapshot for some reason, the best place to get one is another block node.